tests/react/sequential-props.tests.jsx

```diff
@@ -0,0 +1,80 @@
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var should = require('should');
+var Context = {
+  messages: require('../../locales/en-US.json'),
+  router: function() {}
```

Comment:
I tried that, it's not a function that returns an object, it's a function acting as an object. It's weird, but js allows it.